### ISSUE TITLE
Command line returns "Object reference not set to an instance of an object"

### ISSUE THREAD

**Issue by o-o00o-o** [otykier/TabularEditor#240]
So I am trying to run a script at the command line with -S , If I run it interactively it runs fine but if I run it via the command line I get "Object reference not set to an instance of an object"

Is there any way I can get any more clues on where this issue is coming from? It is a 300 line script! 

It calls CustomAction() and I've installed the CustomActions.json in the right place. Any way I can get a line number for the error?

**Comment by o-o00o-o** [comment]
any ideas on this? This has happened since I modularised the script, so I guess it has something to do with that. Repro script

blah.cs (outer)
``` csharp
Info("In Blah2");
var measuresWithAutoDerivatives = Model.AllMeasures
    .Where(m => m.GetAnnotation("AutoDerivatives") != null)
    .ToList();
Info("Blah has " + measuresWithAutoDerivatives.Count.ToString());
CustomAction(measuresWithAutoDerivatives, @"blah2");
```

blah2.cs (inner)
``` csharp
Info("In Blah2");

var measuresWithAutoDerivatives = Selected.Measures
    .Where(m => m.GetAnnotation("AutoDerivatives") != null)
    .ToList();

foreach (Measure measure in measuresWithAutoDerivatives)
{
    Info(measure.Name);
}

```

returns 
```
 Loading custom actions from: C:\Users\bgerhardi\AppData\Local\TabularEditor\CustomActions.json
Dependency tree built in 397 ms
Loading model...
Executing script...
In Blah2
Blah has 4
Script execution error: Object reference not set to an instance of an object.
```

blah2 is the name of the custom action that I have in my CustomActions.json. It feels like Selected doesn't work via console. Perhaps I can somehow make the default behaviour act on all measure if I am in console mode. What is the best way to do this?

**Comment by otykier** [comment]
Fixed in [2.10.0](https://github.com/otykier/TabularEditor/releases/tag/2.10.0)
